tests/e2e/gc_rwx_static_provision.go

```diff
@@ -220,4 +220,156 @@ var _ = ginkgo.Describe("[rwm-csi-guest] File Volume static Provision Test", fun
 		verifyCRDInSupervisor(ctx, f, pod2.Spec.NodeName+"-"+volHandle, crdCNSFileAccessConfig, crdVersion, crdGroup, false)
 	})
 
+	/*
+		Test to verify static volume provision.
+
+		Steps
+		1. Create StorageClass
+		2. Create PVC which uses the StorageClass created in step 1
+		3. Wait for PV to be provisioned
+		4. Wait for PVC's status to become Bound
+		5. Query CNS and check if the PVC entry is pushed into CNS or not
+		6. Create pod using PVC
+		7. Wait for Pod to be up and running and Verify CNSFileAccessConfig CRD is created or not
+		8. Verify Read/Write operation on File volume
+		9. Create a PV using the PVC1 handle
+		10. Create PVC using the PV created above
```

Comment:
To understand step 8 and 9 better, we can mention 
9.Create static PV2 using PVC2 handle
10.Create static PVC2 using PV2 created in step 9

Reply:
Updated as per the comment